modules/core/src/main/resources/schemas/neurosciencegraph/core/selectors/v0.2.0.json

```diff
@@ -0,0 +1,63 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.2.0",
+    {
+      "@base": "{{base}}/schemas/neurosciencegraph/core/selectors/v0.2.0/shapes/"
```

Comment:
We might want to rename it to core/selector/v0.2.0 to have it consistent with other schemas (i.e. schema name singular)?

Reply:
See comment for selectors above.